shared/validation-schemas/src/adoption.ts

```diff
@@ -0,0 +1,27 @@
+import * as Joi from "joi";
+
+export const adoptionValidationSchema = Joi.object({
+  id: Joi.string().required(),
+
+  name: Joi.string().max(30),
+
+  gender: Joi.string().max(6),
+
+  breed: Joi.string().max(30),
+
+  type: Joi.string().max(5),
+
+  age: Joi.number(),
+
+  size: Joi.string().max(6),
+
+  observations: Joi.string().allow("").max(100),
+
+  photos: Joi.array().optional().items(Joi.string()),
```

Comment:
Is it possible to use `Joi.string().uri()` instead?

Reply:
I think so, let's try.